app/models/candidate.rb

```diff
@@ -1,15 +1,19 @@
 # encoding: UTF-8
 class Candidate < ActiveRecord::Base
+  # Associations
   belongs_to :election
   belongs_to :profile
   belongs_to :post
 
+
+  # Validations
   validates :profile_id, uniqueness: {
-    scope: [:post_id, :election_id], message: 'har redan en likadan kandidatur'
-  }, on: :create
+                           scope: [:post_id, :election_id], message: 'har redan en likadan kandidatur'
+                       }, on: :create
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.